src/features/components/atoms/tab/tab-switcher/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { TStepperTab } from '../types'
+import Typography from '../../typography'
+import TabButton from '../../button/tab'
+import FlexBox from '../../flex-box'
+import FlexBoxContainer from '../../flex-box/box'
+import { Localize } from 'components/localization'
+
+const TabSwitcher = ({ tab, onTabClick }: Omit<TStepperTab, 'items'>) => {
+    const button_style = {
+        active: {
+            fontWeight: 'bold',
+            boxShadow: '0px 24px 48px rgba(14, 14, 14, 0.18)',
+            'border-radius': '4px 0px 0px 4px',
+        },
+        inactive: {
+            fontWeight: 'normal',
+            'border-radius': '0px 4px 4px 0px',
+            background: 'transparent',
+        },
+    }
+    return (
+        <FlexBoxContainer
+            direction={'col'}
+            align={'center'}
+            justify={'center'}
+            pt={'20x'}
+            md={{ padding_block: '40x' }}
+        >
+            <Typography.Heading align={'center'} mb={'20x'}>
+                <Localize translate_text={'_t_How to get started with a Deriv EZ account_t_'} />
+            </Typography.Heading>
+            <FlexBox.Box justify={'center'}>
+                <TabButton
+                    style={tab === 'demo' ? button_style['active'] : button_style['inactive']}
```

Comment:
Hi @NikitK-deriv ,
Can we achieve this thing with class  ?

Reply:
Yes @fasihali-deriv but for what? 
Architecture without extra scss files looks cleaner

Reply:
Yes it looks cleaner but is it fine to use JS based styles? Because in new structures @mohsen-deriv is trying to avoid styled component and JS based Styles. This is what i understand in knowledge sharing sessions but if I am wrong then just ignore it 🙏  
`<TabButton  classname={tab === 'demo' ? 'active': 'inactive'}`

and somewhere in scss file you add styles on these classes like. 

`.active{
// css
}`

`.inactive{
// css
}`

Reply:
You're right, I just finished listening. Will refactor it, good catch @fasihali-deriv 👍